setup.py

```diff
@@ -162,23 +163,26 @@ def _collect_requirements(requires):
     "jupyter~=1.0",
     "lxml~=4.6",
     "matplotlib>=3.0.3, <3.4; python_version < '3.10'",  # 3.4.0 breaks holoviews
-    "matplotlib>=3.5, <3.6; python_version == '3.10'",
+    "matplotlib>=3.5, <3.6; python_version >= '3.10'",
     "memory_profiler>=0.50.0, <1.0",
     "moto==1.3.7; python_version < '3.10'",
     "moto==3.0.4; python_version == '3.10'",
+    "moto==4.1.12; python_version == '3.11'",
```

Comment:
Can we also use this `moto` version for Python 3.10? It would be better if we don't have a so many separate pins for the versions..